examples/ConductorSharp.ApiEnabled/Services/TaskExecutionCounterService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using ConductorSharp.Engine.Interface;
+using System.Collections.Concurrent;
+
+namespace ConductorSharp.ApiEnabled.Services
+{
+    public class TaskExecutionCounterService : ITaskExecutionCounterService
+    {
+        private const int _maxTrackedCount = 10000;
+
+        private readonly ConcurrentDictionary<string, RunningTask> _inProgress = new();
+        private readonly ConcurrentDictionary<string, int> _failed = new();
+        private readonly ConcurrentDictionary<string, int> _completed = new();
+
+        public Task OnPolled(RunningTask trackedTask)
```

Comment:
@ognjenkati Are you aware that this will not provide us with exact count? We are ok with that?

Reply:
Removed the count as this was just a example project this is not really required.